catalog/dags/providers/provider_api_scripts/europeana.py

```diff
@@ -157,6 +158,12 @@ def _get_filetype(self, item_data: dict) -> str:
     def _get_filesize(self, item_data: dict) -> int:
         return item_data.get("ebucoreFileByteSize")
 
+    def _get_creator(self, item_data: dict) -> str | None:
+        creators = item_data.get("dcCreator", [])
+        if not creators:
+            return None
+        return creators if isinstance(creators, str) else ", ".join(creators)
```

Comment:
I think it's better to check if the creators is an instance of list or not:
```py
if isinstance(creators, list):
    return ", ".join(creators)
elif isinstance(creators, str):
    return creators
return None
```

Reply:
thank's @obulat for the suggestion I have updated the code to check creators is an instance of list or not. I have also added also tests for some edge cases